tasks/connectors/edgescan/lib/edgescan_location_specifier.rb

```diff
@@ -24,21 +25,81 @@ def location
         end
 
         def to_kenna_asset
-          base = { "external_id" => external_asset_id, "tags" => asset.tags }
+          case type
+          when "url"
+            { **kenna_asset_base, **{ "url" => location }, **host_information(location) }
+          when "hostname"
+            { **kenna_asset_base, **{ "hostname" => location }, **host_information(location) }
+          when "ip"
+            { **kenna_asset_base, **{ "ip_address" => location }, **host_information(location) }
+          when "block"
+            to_kenna_assets_from_block
+          when "cidr"
+            to_kenna_assets_from_cidr
+          end
+        end
+
+        private
+
+        def to_kenna_assets_from_block
```

Comment:
Why are we translating blocks of IP addresses into hosts?

Reply:
It was brought up in the previous meeting that it would be useful to import assets even if there were no vulnerabilities associated with them. If there is no host data and no vulnerability data for a certain location then location specifiers is the only data point that can be used to create a kenna asset.

Reply:
Will this result in the Edgescan connector creating assets in Kenna for hosts that have never been identified by Edgescan, and may not even exist on the network?

Reply:
Removed in ffb6b20.